components/pager/overview.md

```diff
@@ -0,0 +1,118 @@
+1. Add the `TelerikPager` tag
+1. (optional) Populate it's `Total` parameter
+1. (optional) Set it's `ButtonCount` parameter
+1. (optional) Set the `PageChanged` parameter
+1. (optional) Set `Page` or `@bind-Page` (one and two-way data binding)
+1. (optional) Set a `PageSize`
+
+>caption Basic setup of Pager component in your application
+
+````CSHTML
+@*Basic Pager configuration.*@
+
+<TelerikPager Total="TotalItems"
+              ButtonCount="ButtonCount"
+              PageSize="ItemsOnPage"
+              @bind-Page="CurrentPage">
+
+</TelerikPager>
+
+@code {
+    public int TotalItems { get; set; } = 80;
+    public int ButtonCount { get; set; } = 4;
+    public int ItemsOnPage { get; set; } = 10;
+    public int CurrentPage { get; set; } = 2;
+}
+````
+
+>caption The result from the code snippet above
+
+![basic configuration of the pager](images/pager-basic-configuration-screenshot.jpg)
+
+## Features
+* `Class` - The CSS class that will be rendered on the main wrapping element of the Pager.
+* `Total` - **int** - Represents the total count of items in the pager.
+* `ButtonCount` - **int** - The number of pages to be visible. To take effect the `ButtonCount` must be **less** than the pages count (ButtonCount < Total / number of items on the page)
+* `Page` and `@bind-Page` - **int** - Represents the current page of the pager. Those parameters are respectively for one and two-way data binding. If no `Page` or `@bind-Page` are provided they will default to the first page (1).
+* `PageChanged` - Fires when a new page is selected (used in one-way data binding).
+* `PageSize` - **int** - The number of items to be presented on a pages.
+
+## Examples
+
+>caption Observe the behavior of the Pager with one-way data binding
+
+````CSHTML
+@*This example showcases the usage of Page and PageChanged in conjunction*@
+
+<TelerikPager Total="TotalItems"
+              ButtonCount="ButtonCount"
+              PageSize="ItemsOnPage"
+              Page="CurrentPage"
+              PageChanged="@( (int page) => PageChangedHandler(page)  )">
+
+</TelerikPager>
+
+<div class="text-info">@Result</div>
+
+@code {
+    public int TotalItems { get; set; } = 80;
+    public int ButtonCount { get; set; } = 4;
+    public int ItemsOnPage { get; set; } = 10;
+    public int CurrentPage { get; set; } = 2;
+    public string Result { get; set; } = String.Empty;
+
+    void PageChangedHandler(int page)
+    {
+        CurrentPage = page;
+        Result = $"Current page: {page}";
+    }
+}
+````
+>caption The result from the code snippet above
+
+![config of the pager with one-way binding](images/pager-data-binding.gif)
+
+>caption Observe the behavior of the Pager with two-way data binding
+
+````CSHTML
+@*This example showcases the usage of Page and PageChanged in conjunction*@
+
+<TelerikPager Total="TotalItems"
+              ButtonCount="ButtonCount"
+              PageSize="ItemsOnPage"
+              @bind-Page="CurrentPage">
```

Comment:
For two-way binding the example should show not only how the component puts the page in an external variable, but also how changing the variable affects the pager. For example, bind it to a NumericTextbox as well so people can see how the pager reacts to that as well.

Reply:
Done